DbcParserLib.Tests/DbcParserLib.Tests.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
I think this is failing because ci.yml is setting up the environment with a .NET 6

Reply:
I generally didnt know that this was setup in the commit files. Wasnt actions once only to setup in github itself?